src/bot.js

```diff
@@ -0,0 +1,99 @@
+import { PIXELS_PER_TILE } from './constants';
+import { getTraversableCells } from './helpers/map';
+import {
+  findMyEndzone,
+  findEnemyEndzone,
+  findFlagStation,
+  findEnemy,
+  findEnemyFC,
+} from './helpers/finders';
+import { myTeamHasFlag, enemyTeamHasFlag } from './helpers/gameState';
+import { getMe } from './helpers/player';
+import getTarget from './helpers/path';
+import { move } from './utils';
+
+/*
+ * The logic/flowchart to get where our goal is.
+ *   If I have the flag, go to my endzone.
+ *   If an enemy in view has the flag, chase him.
+ *   If the enemy team has the flag but I can't see them, go to their endzone.
+ *   If we have the flag, go to our endzone.
+ *   Else, go to the flag station.
+ */
+function getGoal() {
+  let goal;
+  const me = getMe();
+  // If the bot has the flag, go to the endzone
+  if (me.flag) {
+    const chaser = findEnemy();
+    if (false && chaser) { // eslint-disable-line no-constant-condition
+      // Really bad jukes !!!!! DISABLED FOR NOW
```

Comment:
I know this wasn't your code @billmwong , but we should address it at some point (maybe this PR?)